libmscore/tremolo.cpp

```diff
@@ -431,8 +401,10 @@ void Tremolo::layoutTwoNotesTremolo(qreal x, qreal y, qreal h, qreal _spatium)
             }
       else {
             firstChordStaffY = _chord1->pagePos().y() - _chord1->y();  // y coordinate of the staff of the first chord
-            y1 = _chord1->stemPosBeam().y() - firstChordStaffY + _chord1->defaultStemLength();
-            y2 = _chord2->stemPosBeam().y() - firstChordStaffY + _chord2->defaultStemLength();
+            const std::array<double, 2> extendedLen 
+               = extendedStemLenWithTwoNoteTremolo(this, spatium(), _chord1->defaultStemLength(), _chord2->defaultStemLength());
+            y1 = _chord1->stemPos().y() - firstChordStaffY + extendedLen[0];
+            y2 = _chord2->stemPos().y() - firstChordStaffY + extendedLen[1];
```

Comment:
In case anyone is wondering, regarding the `stemPosBeam()` to `stemPos()` change: this is intended. We're actually getting the y coordinate of stem tip here by calculating stem position (which is where it's attached to a note) and stem length. `stemPosBeam()` gives the position of the nearest note to beam, where as `stemPos()` gives the real start position of the stem (the position of the furthest note to beam). So `stemPosBeam()` is actually a bad mistake.